redbeat/schedulers.py

```diff
@@ -48,6 +48,28 @@
 CELERY_4_OR_GREATER = CELERY_VERSION[0] >= 4
 REDIS_3_OR_GREATER = REDIS_VERSION[0] >= 3
 
+# Copy from:
```

Comment:
Is this a direct copy or was it modified? if the later let's call out what was changed and why to make future maintenance easier.

Reply:
It was modified and I changed the comment.